readme/configuration-management.md

```diff
@@ -42,6 +52,45 @@ The solution is to make sure that the referenced content exists before the featu
 * Use the default_content module to export the referenced content as JSON files, and store these files with your feature or in a dependency.
```

Comment:
These two lines should be updated to be like "store these files with a custom feature or module"—basically genericize the features language.